python_scripts/metrics_classification.py

```diff
@@ -304,6 +304,15 @@
     classifier, data_test, target_test, pos_label='donated',
     marker="+"
 )
+disp = PrecisionRecallDisplay.from_estimator(
+    dummy_classifier, data_test, target_test, pos_label='donated',
+    color="tab:orange", linestyle="--", ax=disp.ax_)
+plt.xlabel("Recall                  ")
+plt.text(0.45, -0.2, "=  TPR or sensitivity", size=8)
```

Comment:
I find it too small and it could be tricky if someone has copied the `rc` file or not. Could we just instead add it in the label between bracket.

```python
plt.xlabel("Recall (also known as TPR or sensitivity)")
```

Reply:
Done :)